packages/cms-cli/commands/lint.js

```diff
@@ -27,51 +27,83 @@ const {
 } = require('../lib/usageTracking');
 
 const COMMAND_NAME = 'lint';
+const DESCRIPTION = 'Lint a file or folder for HubL syntax';
 
-function configureCommand(command) {
-  command
-    .version(version)
-    .description('lint a file or folder for HubL syntax')
-    .arguments('<path>')
-    .action(async filepath => {
-      setLogLevel(command);
-      logDebugInfo(command);
-      const { config: configPath } = command;
-      loadConfig(configPath);
-      checkAndWarnGitInclusion();
+const action = async (args, options) => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
 
-      if (!(validateConfig() && (await validatePortal(command)))) {
-        process.exit(1);
-      }
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
 
-      const portalId = getPortalId(command);
+  const portalId = getPortalId(options);
 
-      trackCommandUsage(COMMAND_NAME, {}, portalId);
+  trackCommandUsage(COMMAND_NAME, {}, portalId);
 
-      filepath = resolveLocalPath(filepath);
+  const localPath = resolveLocalPath(args.localPath);
+  const groupName = `Linting "${localPath}"`;
 
-      const groupName = `Linting "${filepath}"`;
-      logger.group(groupName);
-      let count = 0;
-      try {
-        await lint(portalId, filepath, result => {
-          count += printHublValidationResult(result);
-        });
-      } catch (err) {
-        logger.groupEnd(groupName);
-        logErrorInstance(err, { portalId });
-        process.exit(1);
-      }
-      logger.groupEnd(groupName);
-      logger.log('%d issues found', count);
+  logger.group(groupName);
+  let count = 0;
+  try {
+    await lint(portalId, localPath, result => {
+      count += printHublValidationResult(result);
     });
+  } catch (err) {
+    logger.groupEnd(groupName);
+    logErrorInstance(err, { portalId });
+    process.exit(1);
+  }
+  logger.groupEnd(groupName);
+  logger.log('%d issues found', count);
+};
+
+const configureYargs = {
```

Comment:
I did this and the `configureCommander` below just to make the division a bit more clear until we are able to remove commander

Reply:
TBH, I am not a huge fan of the extra objects when it comes to readability but understand the intent. I also think it would be easier to clean up when we remove Commander.js if we didn't have to touch the yargs code. What if instead we used a comment for the commander helpers?

Reply:
Sure, I think it still makes sense to wrap do the `configureYargs` object, and one the migration is done we can just export that whole object.  I can unwrap commander and just use a comment for it.  You fine if I keep the yargs stuff grouped?

Reply:
It is actually the yargs grouping that I find awkward because it adds what feels like an unnecessary layer of indirection — over my career I have tended towards avoiding abstractions whenever possible. In this case, I think developers adding commands will need to understand the `yargs` command module interface, and once the migration to `yargs` is complete, I don't think we will want to export things other than what `yargs` expects so that if `yargs` adds additional properties we don't hit conflicts.

Reply:
I guess I'm not following.  I don't see how 

```js
const configureYargs = {,
   command: ...
   describe: ...
   ...
};

module.exports = configureYargs;
```

or even

```js
module.exports = {
   command: ...
   describe: ...
   ...
}
```

in the future after commander is removed would be much different than

```js
exports.command = ...;
exports.describe = ...;
...
```

To me, it is more clear, and not really an abstraction to do the first or second option, since it inherently shows what Yargs needs exported, while keeping the other functions in the file (.e.g `async function action`) are not something Yargs is expecting to have exported.

The other option would need to be something like the following until we remove commander

```js
// Yargs Exports
exports.command = ...;
exports.describe = ...;
...
...
...
...
// End Yargs Exports
```

I guess my overall point being is I don't see a big difference between `module.exports = {...}` vs `exports.command = ...; exports.describe = ...;`  We seem to do a combo of both from what I have seen

Reply:
I am fine with `module.exports` but I'd rather we compose the object directly than spread over another object.

Reply:
I also don't think this is that big of a deal if you prefer the way it is written. There are bigger things to tackle like finishing up the migration to `yargs`, and I don't want to get too bogged down bikeshedding.

Reply:
I agree :) I'll go with the composing module.exports directly